demo/blocks_kitchen_sink/run.py

```diff
@@ -0,0 +1,233 @@
+import gradio as gr
+import time
+from os.path import abspath, join, pardir
+
+KS_FILES = abspath(join(pardir, "kitchen_sink", "files"))
```

Comment:
Nit but I don't think this is the absolute path? I have to be in this directory to be able to run this.